components/create-bounty-modal.tsx

```diff
@@ -172,7 +172,7 @@ export default function CreateBountyModal() {
         decimals: transactionalERC20?.decimals,
         amount: rewardAmount,
         setAmount: setRewardAmount,
-        tokens: customTokens.filter(token => token?.isTransactional !== true),
+        tokens: customTokens.filter(token => token?.isReward !== false),
```

Comment:
isn't it the same thing?

Reply:
if there is no allowed in the registry with this condition, it can take any token that exists in our database.

Reply:
It could be simply `token?.isReward` or a double negation `!!token?.isReward`, as we want to know if it is `true`